pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
```

Comment:
Yes, a new combination of labels is a new time series in Prometheus. There could be is that whenever a Pod is created and the `NodeName` is _not yet_ set, then there will be two time-series once it is set, and the time-series where the `NodeName` is set will be missing the first scrape(s) (depending on when it is set). I would say this is fine for now, we should just keep it in mind.